cime/scripts/Tools/Makefile

```diff
@@ -298,14 +301,6 @@ ifeq ($(findstring -DLINUX,$(CPPDEFS)),-DLINUX)
   endif
 endif
 
-# Atleast on Titan+cray mpi, MPI_Irsends() are buggy, causing hangs during I/O
-# Force PIO to use MPI_Isends instead of the default, MPI_Irsends
```

Comment:
I thought we wanted to keep this.  @jayeshkrishna ?

Reply:
This was the PR that changed this: https://github.com/ESMCI/cime/pull/3502

Reply:
JimE added him as a reviewer on that PR and he didn't object, so I'm going to assume he's OK with these changes unless he says something.

Reply:
Yes we can assume the SCORPIO build will now handle this.

Reply:
Yes, this was handled in Scorpio (https://github.com/E3SM-Project/scorpio/pull/301) and Scorpio classic (https://github.com/E3SM-Project/scorpio/pull/300). So this change in CIME is ok.